app/code/Magento/Catalog/Model/Category.php

```diff
@@ -317,16 +313,6 @@ protected function getCustomAttributesCodes()
         return $this->getCustomAttributeCodes->execute($this->metadataService);
     }
 
-    /**
-     * @throws \Magento\Framework\Exception\LocalizedException
-     * @return \Magento\Catalog\Model\ResourceModel\Category
-     * @deprecated because resource models should be used directly
-     */
-    protected function _getResource()
-    {
-        return parent::_getResource();
-    }
-
```

Comment:
same as with `$_resource`. It's not super important but a minor improvement that I made while I was at it. Should it be a separate PR now that there are no other changes in the file anymore?